### PR TITLE
Fix vtk output bugs

### PR DESCRIPTION
**Pull request by atgeirr**

### CODE REVIEW

File: opm/models/io/vtkmultiwriter.hh [hunk 1]
```diff
@@ -85,9 +85,11 @@ class VtkMultiWriter : public BaseOutputWriter
 
             // determine name to write into the multi-file for the
             // current time step
+            // The file names in the pvd file are relative, the path should therefore be stripped.
+            const auto localFileName = fileName.substr(fileName.find_first_of('/') + 1);
```
Comment (akva2) [inline opm/models/io/vtkmultiwriter.hh:89]: i'd use the shiny filesystem class for this. it would stop me from wondering 'why the first slash'.

Comment (atgeirr) [reply opm/models/io/vtkmultiwriter.hh:89]: Ok, what is the best way to get the local filename from the string?

Comment (akva2) [reply opm/models/io/vtkmultiwriter.hh:89]: https://en.cppreference.com/w/cpp/filesystem/path - relative.

Comment (atgeirr) [reply opm/models/io/vtkmultiwriter.hh]: Thanks. In the end I went with the filename() function for simplicity, as the output directory is the same.

**Comment by atgeirr** [comment]
jenkins build this please

**Comment by atgeirr** [comment]
It turns out that the local tests here combined the vtk output code with fluid systems that did not have the phaseIsActive() function. Resolved by adding trivial such function to the base fluid system upstream.

**Comment by atgeirr** [comment]
jenkins build this opm-material=420 please

**Comment by atgeirr** [comment]
I will take a chance and self-merge this and the upstream PR. The changes are small and simple, review comments have been incorporated, and we solve a real problem with the VTK output.

**Pull request merged**